octokit/service.go

```diff
@@ -0,0 +1,25 @@
+package octokit
+
+import (
+	"net/url"
+)
+
+type GenericService struct {
```

Comment:
I'd like to flesh out our patterns across a couple of services without worrying about DRYing the common parts up just yet.